src/Serialization.purs

```diff
@@ -0,0 +1,187 @@
+module Serialization
+  ( convertTransaction
+  , addressPubKeyHash
+  , convertBigInt
+  , convertTxOutput
+  , newAddressFromBech32
+  , newAssetName
+  , newTransactionHash
+  , toBytes
+  , newBaseAddressFromAddress
+  , module X
+  )
+where
+
+import Data.BigInt as BigInt
+import Data.Maybe (Maybe(..))
+import Data.Newtype (unwrap)
+import Data.Traversable (traverse_, for_)
+import Data.FoldableWithIndex (forWithIndex_)
+import Data.UInt (UInt)
+import Effect (Effect)
+import Prelude
+import Serialization.Types (Address, AssetName, Assets, AuxiliaryData, BaseAddress, BigNum, DataHash, Ed25519KeyHash, Ed25519Signature, MultiAsset, PlutusScript, PlutusScripts, PublicKey, ScriptHash, StakeCredential, Transaction, TransactionBody, TransactionHash, TransactionInput, TransactionInputs, TransactionOutput, TransactionOutputs, TransactionWitnessSet, Value, Vkey, Vkeywitness, Vkeywitnesses)
```

Comment:
These two lines are a little long

Reply:
Oh, also are you hiding certain imports from `Serialization.Types` to avoid re-exporting them? If so, I would personally prefer to use `hiding` in that case

Reply:
Good catch, I probably just added a new type and forgot to re-import it here. I think we can drop this re-export.